go.mod

```diff
@@ -3,15 +3,9 @@ module github.com/crossplane-contrib/provider-secret
 go 1.13
 
 require (
-	github.com/apex/log v1.9.0 // indirect
```

Comment:
note: these changes were able to sneak in because the gh action wasn't run on the PR in which it was introduced. Future PRs will fail on untidy go.mod